config/config_helpers.go

```diff
@@ -173,6 +175,37 @@ func getPlatform(trackInclude *TrackInclude, terragruntOptions *options.Terragru
 	return runtime.GOOS, nil
 }
 
+// Return the path from the repository root
+func getPathFromRepoRoot(trackInclude *TrackInclude, terragruntOptions *options.TerragruntOptions) (string, error) {
+	repoAbsPath, err := shell.GitTopLevelDir(terragruntOptions, terragruntOptions.WorkingDir)
+	if err != nil {
+		return "", errors.WithStackTrace(err)
+	}
+
+	pathFromRoot := strings.TrimSpace(strings.Replace(
+		strings.TrimSpace(terragruntOptions.WorkingDir),
+		strings.TrimSpace(string(repoAbsPath))+"/",
+		"", -1,
+	))
```

Comment:
I think the following should work, and is more clear what is going on:
```suggestion
	return filepath.Rel(repoAbsPath, terragruntOptions.WorkingDir)
```

Reply:
See https://go.dev/play/p/eClbMKXPlqu